dotNETStandard/HealthVaultProviderManagementPortal/HealthVaultProviderManagementPortal/Controllers/PatientController.cs

```diff
@@ -194,5 +251,13 @@ private static JsonSerializerSettings GetDeserializationSettings()
 
             return settings;
         }
+
+        private static JsonSerializerSettings GetSerializationSettings()
```

Comment:
Well, I have a static variable that I set at the top of the file using these functions